    <?xml version="1.0" encoding="UTF-8"?>
    <ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
    <property name="geometry">
    <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>500</height>
    </rect>
    </property>
    <property name="windowTitle">
    <string>Form</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
        <rect>
        <x>10</x>
        <y>10</y>
        <width>381</width>
        <height>80</height>
        </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
        <item>
            <widget class="QLineEdit" name="le_view">
        <property name="font">
        <font>
            <pointsize>40</pointsize>
        </font>
        </property>
        <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
        </property>
        </widget>
        </item>
    </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
        <rect>
        <x>10</x>
        <y>100</y>
        <width>381</width>
        <height>391</height>
        </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="2">
        <widget class="QPushButton" name="btn_6">
        <property name="minimumSize">
        <size>
            <width>0</width>
            <height>80</height>
        </size>
        </property>
        <property name="text">
        <string>6</string>
        </property>
        </widget>
        </item>
        <item row="1" column="3">
        <widget class="QPushButton" name="btn_mul">
        <property name="minimumSize">
        <size>
            <width>0</width>
            <height>80</height>
        </size>
        </property>
        <property name="text">
        <string>Ã—</string>
        </property>
        </widget>
        </item>
        <item row="3" column="0">
        <widget class="QPushButton" name="btn_c">
        <property name="minimumSize">
        <size>
            <width>0</width>
            <height>80</height>
        </size>
        </property>
        <property name="text">
        <string>C</string>
        </property>
        </widget>
        </item>
        <item row="1" column="1">
        <widget class="QPushButton" name="btn_5">
        <property name="minimumSize">
        <size>
            <width>0</width>
            <height>80</height>
        </size>
        </property>
        <property name="text">
        <string>5</string>
        </property>
        </widget>
        </item>
        <item row="0" column="0">
        <widget class="QPushButton" name="btn_7">
        <property name="minimumSize">
        <size>
            <width>0</width>
            <height>80</height>
        </size>
        </property>
        <property name="text">
        <string>7</string>
        </property>
        </widget>
        </item>
        <item row="1" column="0">
        <widget class="QPushButton" name="btn_4">
        <property name="minimumSize">
        <size>
            <width>0</width>
            <height>80</height>
        </size>
        </property>
        <property name="text">
        <string>4</string>
        </property>
        </widget>
        </item>
        <item row="0" column="2">
        <widget class="QPushButton" name="btn_9">
        <property name="minimumSize">
        <size>
            <width>0</width>
            <height>80</height>
        </size>
        </property>
        <property name="text">
        <string>9</string>
        </property>
        </widget>
        </item>
        <item row="0" column="3">
        <widget class="QPushButton" name="btn_div">
        <property name="minimumSize">
        <size>
            <width>0</width>
            <height>80</height>
        </size>
        </property>
        <property name="text">
        <string>/</string>
        </property>
        </widget>
        </item>
        <item row="0" column="1">
        <widget class="QPushButton" name="btn_8">
        <property name="minimumSize">
        <size>
            <width>0</width>
            <height>80</height>
        </size>
        </property>
        <property name="text">
        <string>8</string>
        </property>
        </widget>
        </item>
        <item row="2" column="0">
        <widget class="QPushButton" name="btn_1">
        <property name="minimumSize">
        <size>
            <width>0</width>
            <height>80</height>
        </size>
        </property>
        <property name="text">
        <string>1</string>
        </property>
        </widget>
        </item>
        <item row="2" column="1">
        <widget class="QPushButton" name="btn_2">
        <property name="minimumSize">
        <size>
            <width>0</width>
            <height>80</height>
        </size>
        </property>
        <property name="text">
        <string>2</string>
        </property>
        </widget>
        </item>
        <item row="2" column="2">
        <widget class="QPushButton" name="btn_3">
        <property name="minimumSize">
        <size>
            <width>0</width>
            <height>80</height>
        </size>
        </property>
        <property name="text">
        <string>3</string>
        </property>
        </widget>
        </item>
        <item row="2" column="3">
        <widget class="QPushButton" name="btn_add">
        <property name="minimumSize">
        <size>
            <width>0</width>
            <height>80</height>
        </size>
        </property>
        <property name="text">
        <string>+</string>
        </property>
        </widget>
        </item>
        <item row="3" column="1">
        <widget class="QPushButton" name="btn_0">
        <property name="minimumSize">
        <size>
            <width>0</width>
            <height>80</height>
        </size>
        </property>
        <property name="text">
        <string>0</string>
        </property>
        </widget>
        </item>
        <item row="3" column="2">
        <widget class="QPushButton" name="btn_result">
        <property name="minimumSize">
        <size>
            <width>0</width>
            <height>80</height>
        </size>
        </property>
        <property name="text">
        <string>=</string>
        </property>
        </widget>
        </item>
        <item row="3" column="3">
        <widget class="QPushButton" name="btn_min">
        <property name="minimumSize">
        <size>
            <width>0</width>
            <height>80</height>
        </size>
        </property>
        <property name="text">
        <string>-</string>
        </property>
        </widget>
        </item>
    </layout>
    </widget>
    </widget>
    <resources/>
    <connections/>
    </ui>
